pkgs/servers/home-assistant/custom-components/prometheus-query/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+}:
+
+let
+  pname = "prometheus_sensor";
+  version = "1.0.0";
+in
+stdenv.mkDerivation {
```

Comment:
Shall we try to tackle `buildHomeAssistantCustomComponent` in a separate PR?

Reply:
No, it would be nice to have this from the get go.

Reply:
Ok, I'll try to add some changes here in the next couple of days.

Reply:
PING @graham33, would be nice to get this in some time soon.